cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerConfigurationDiff.java

```diff
@@ -111,10 +112,11 @@ private boolean isEntryReadOnly(ConfigEntry entry) {
     }
 
     /**
-     * @return A map which can be used for dynamic configuration of kafka broker
+     * Adds an entry to a map which can be used for dynamic configuration of kafka broker
+     * @param updatedConfig map to add an entry
      */
-    public Map<ConfigResource, Collection<AlterConfigOp>> getConfigDiff() {
-        return Collections.singletonMap(Util.getBrokersConfig(brokerId), diff);
+    public void addConfigDiff(Map<ConfigResource, Collection<AlterConfigOp>> updatedConfig) {
```

Comment:
I'm not sure I understand the naming and the Javadoc here. If you add something to the map, it is not updated yet so the parameter name is wrong? I'm also not sure why a separate method is needed here. It looks like any call for this method can be done in single line.